gradlew

```diff
@@ -1,7 +1,7 @@
-#!/usr/bin/env sh
+#!/bin/sh
 
 #
-# Copyright 2015 the original author or authors.
+# Copyright © 2015-2021 the original authors.
 #
 # Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
wow, so many changes to this file! Is this something that you write by hand or is it automatically created? What caused the changes?

Reply:
No this file is auto-generated by gradle